avalon/io.py

```diff
@@ -138,6 +138,14 @@ def _from_environment():
             # Optional path to scenes directory (see Work Files API)
             ("AVALON_SCENEDIR", None),
 
+            # Optional hierarchy for the current Asset. This can be referenced
+            # as `{hierarchy}` is your file templates.
```

Comment:
Found a typo :P
`# as {hierarchy} is your file templates.`
Should be
`# as {hierarchy} i*n* your file templates.`

Reply:
Oh dear, so simple and still I need several commits... :flushed: :slightly_smiling_face: